docs/upgrading.md

```diff
@@ -0,0 +1,111 @@
+# Upgrading Clear Containers installations
+
+This document discusses the available options for upgrading a system running Intel® Clear Containers.
+
+## Terminology
+
+There are currently two implementations of Clear Containers:
+
+- Clear Containers 2.x (or just 2.x) refers to the previous implementation
+  of Clear Containers which uses a runtime called `cc-oci-runtime` and
+  whose code is available at: https://github.com/01org/cc-oci-runtime
+
+- Clear Container 3.x (or just 3.x) refers to the current implementation
+  which uses a runtime called `cc-runtime` and whose code is available
+  at: https://github.com/clearcontainers/runtime
+
+## Maintenance warning
+
+The 2.x codebase is no longer being developed. New releases will _only_ be
```

Comment:
nitpick: MD query: here you use _underbars_ which come out as italic, but below you use *stars* that render the same - are they notionally different?

Reply:
Fixing...